graspy/match/gmp.py

```diff
@@ -136,6 +138,11 @@ def __init__(
         else:
             msg = '"gmp" must be a boolean'
             raise TypeError(msg)
+        if (n_jobs > 0 and type(n_jobs) is int) or n_jobs == -1:
```

Comment:
remove this stuff, n_jobs can be -2, -3, etc. I think I am fine with just letting joblib to the input checking

Reply:
> remove this stuff, n_jobs can be -2, -3, etc. I think I am fine with just letting joblib to the input checking

Sure